src/ape/pytest/runners.py

```diff
@@ -83,7 +83,7 @@ def pytest_exception_interact(self, report, call):
                 capman.resume_global_capture()
 
     @pytest.hookimpl(hookwrapper=True)
-    def pytest_runtest_protocol(self, item, nextitem):
+    def pytest_runtest_call(self, item):
```

Comment:
Note, this seems to work with more complex contract deployments using `session` scope, which is awesome!